k8s/upgrades/0.9.0-1.0.0/label_patch.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+ns=$1
+## Below snippet will remove the openebs.io/version label from
+## deployment.spec.selector.matchLabels
+
+## Remove openebs.io/version from maya-apiserver
+## Get maya-apiserver deployment name
+maya_deploy_name=$(kubectl get deploy \
```

Comment:
Need to check the error while getting deployment name also

Reply:
we will have a check in other places also while getting deployment name

Reply:
done